src/Illuminate/Support/Carbon.php

```diff
@@ -2,47 +2,8 @@
 
 namespace Illuminate\Support;
 
-use JsonSerializable;
 use Carbon\Carbon as BaseCarbon;
-use Illuminate\Support\Traits\Macroable;
 
-class Carbon extends BaseCarbon implements JsonSerializable
+class Carbon extends BaseCarbon
 {
-    use Macroable;
```

Comment:
This is a breaking change. Either send it to v5.7, or don't remove `Macroable`.

Reply:
No, it's not since compatible methods are now in Carbon itself.

Reply:
You're saying that peoples existing macros won't break with this change?